src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
```

Comment:
The Circle class should not directly import the Figure class if it's not being used. Make sure to remove unused imports to keep the code clean.